flowmachine/flowmachine/features/utilities/event_table_subset.py

```diff
@@ -94,6 +103,10 @@ def __init__(
                 )
         self.columns = sorted(self.columns)
```

Comment:
Does it make sense to have `self.sqalchemy_columns` here instead of line 183 like you do with `self.sqlachemy_table` or is it overkill?

Reply:
I actually moved it here temporarily but in the end decided to move it back. This doesn't feel ideal yet but it's probably better to refactor it further in a separate PR.